pkgs/development/web/netlify-cli/default.nix

```diff
@@ -1,17 +1,41 @@
-{ callPackage, fetchFromGitHub, lib, pkgs }:
-let
-  nodePackages = import ./composition.nix { inherit pkgs; };
-  sourceInfo = (lib.importJSON ./netlify-cli.json);
-in
-  nodePackages.package.override {
-    preRebuild = ''
-      export ESBUILD_BINARY_PATH="${pkgs.esbuild_netlify}/bin/esbuild"
-    '';
-    src = fetchFromGitHub {
-      inherit (sourceInfo) owner repo rev sha256;
-    };
-    bypassCache = true;
-    reconstructLock = true;
-    passthru.tests.test = callPackage ./test.nix { };
-    meta.maintainers = with lib.maintainers; [ roberth ];
```

Comment:
I think we can carry over this maintainer

Reply:
I won't be an _active_ maintainer of this package, as I don't really do much js packaging.
I'm not a fan of complex fixed output derivations, but I can see that it may solve a problem as a stop gap until RFC 92 is implemented and adopted, which will take a while.
So I won't spend much time on this package, but I can answer questions, ask contributors for tests and hit merge.